aws-toolkit-eclipse-master/bundles/com.amazonaws.eclipse.lambda/src/com/amazonaws/eclipse/explorer/lambda/FunctionLogsTable.java

```diff
@@ -266,7 +266,10 @@ public void run() {
                 String text = CloudWatchLogsUtils.convertLogEventsToString(events);
 
                 File file = File.createTempFile(logGroupName, ".txt");
-                IOUtils.copy(new StringInputStream(text), new FileOutputStream(file));
+                try (final StringInputStream inStream = new StringInputStream(text);
+                     final FileOutputStream = new FileOutputStream(file)) {
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

Syntax error on token &#34;final&#34;, float expected